eogrow/core/schemas.py

```diff
@@ -79,6 +69,9 @@ class PipelineSchema(BaseSchema):
     raise_on_temporal_mismatch: bool = Field(
         False, description="Treat `TemporalDimensionWarning` as an exception during EOExecution."
     )
+    debug: bool = Field(
+        False, description="Run pipeline with less parallelization to enable coupling with the debugger."
```

Comment:
```suggestion
        False, description="Run pipeline without parallelization to enable debugging."
```

Reply:
well....
it's not true :see_no_evil: it only disable parallelization of the eoexecutor...

Reply:
Updated, hope it's better

Reply:
not entirely accurate but whatever, accepted